ga4gh/cli.py

```diff
@@ -266,10 +269,10 @@ def server_main(parser=None):
             description="GA4GH reference server")
     addGlobalOptions(parser)
     args = parser.parse_args()
-    frontend.configure(args.config_file, args.config)
+    frontend.configure(args.config_file, args.config, args.oidc_config_file)
     frontend.app.run(
-        host="0.0.0.0", port=args.port,
-        use_reloader=not args.dont_use_reloader)
+        host="localhost", port=args.port,
+        use_reloader=not args.dont_use_reloader, ssl_context="adhoc")
```

Comment:
I don't think we want to force SSL all the time, since this is a local server for development. Ideally we want a CLI option to enable and disable SSL. For now, turning SSL on when the `oidc_config_file` is present and off otherwise is fine I think.